js/main.js

```diff
@@ -0,0 +1,23 @@
+function randomNumber(min, max) {
+  if ((min < 0) || (max < 0) || (min === max)) {
+    return -1;
+  } else if (min > max) {
+    [min, max] = [max, min];
+  }
+  const r = Math.random() * (max - min + 1) + min;
+  return Math.floor(r);
```

Comment:
Здесь можно обойтись одной строкой
```suggestion
  return Math.floor(Math.random() * (max - min + 1) + min);
```